<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fortosi_selClass</class>
 <widget class="QWidget" name="fortosi_selClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Φόρτωση</string>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>3.Αποστολή Τελευταίας Φόρτωσης</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>291</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>1.Νέα Φόρτωση</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>281</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>2.Φόρτωση σε εξέλιξη</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
